pkg/resolution/common/interface_v2.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2023 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+
+	pipelinev1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+)
+
+// ResolverPayload is the struct which holds the payload to create
+// the Resolution Request CRD.
+type ResolverPayload struct {
+	Params pipelinev1.Params
```

Comment:
Why not `*v1beta1.ResolutionRequestSpec` to match the impl?

Reply:
done

Reply:
fixed